code/modules/buildmode/buildmode.dm

```diff
@@ -22,6 +22,8 @@
 	// dirswitch UI
 	var/atom/movable/screen/buildmode/bdir/dirbutton
 	var/list/dirswitch_buttons = list()
+	// item preview for selected item
```

Comment:
```suggestion
	/// item preview for selected item
```

dmdoc is better, wish the rest of the datum had it but at the very least the new stuff should